virttest/qemu_qtree.py

```diff
@@ -471,7 +471,7 @@ def check_guests_proc_scsi(self, info):
         scsis = set()
         for scsi in _scsis:
             # Ignore IDE disks
-            if scsi[5] != 'CD-ROM':
+            if scsi[5] != 'CD-ROM' and scsi[4] == 'QEMU':
```

Comment:
Can you add comments will skip qemu scsi device ?

Reply:
@xutian, thanks for your review, xu. I have added the comment. Can you help to check again? Any confusion, please comment, thank you.